### PR TITLE
Break code in small components

### PR DESCRIPTION
**Pull request by helderburato**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -58,15 +33,15 @@ app.get("/v1/:username", async (req, res, next) => {
     cache.put(key, data, 1000 * 3600); // Store for an hour
     res.json(data);
   } catch (err) {
-    next(new VError(err, "Visiting the profile has failed."));
+    next(new VError(err,  alerts.error.profileDisabled));
```
[the remark this answers is not in the document]
Comment (helderburato) [reply src/index.js]: Not yet, but.. installing right now 👍

**Comment by helderburato** [comment]
I made some changes:
- Remove: unused `base64data` param from module util/twitter.js;
- Fix: formatted all files using prettier.

@sallar could u check, please? :)

**Comment by sallar** [comment]
Thanks @helderburato

**Comment by helderburato** [comment]
You're welcome! 💯

**Pull request merged**